07-multi-hop/example_test_guide.md

```diff
@@ -0,0 +1,113 @@
+_The following is a possible way to run the tests, this is just an example_
+## Goal: Check multi-hop connectivity over IPv6
+
+### Pre-setup Steps
+1. Book n>15 iotlab M3 nodes (preferably in grenoble) in a straight line 3 nodes apart. (so x--x--x--x)
+2. Add the following CFLAGS to the `tests/gnrc_udp`:
+```
+CFLAGS += -DGNRC_NETIF_IPV6_ADDRS_NUMOF=15 \
```

Comment:
You *don't* need 15 addresses **per node**! You need at most 2 and that is already preconfigured.

Reply:
The reasoning behind this is it will still compile/function and if a whole bunch of nodes are booked rpl can still work (even if many nodes are in range).  This means that we don't have to access each node for blacklisting.  If you would like I can say "set it to an appropriate number" so that it doesn't seem so specific.